refinery/data_set_manager/tasks.py

```diff
@@ -273,22 +276,26 @@ def parse_isatab(username, public, path, identity_id=None,
         return data_set_uuid
 
 
-@task()
-def generate_auxiliary_file(auxiliary_node, parent_node_file_store_item):
+@task(soft_time_limit=3600)
```

Comment:
Since this is a file-based operation (we have to both move a file and do an operation on it), the timeout should match the FileImportTask's timeout

Reply:
The FileImportTask timeout was chosen to accommodate downloads from sites on public Internet which can take a really long time (e.g., from ftp://ftp.sra.ebi.ac.uk).
Here we are dealing with transfers to/from S3 within AWS network. It would be great to benchmark how long does this operation take for a typical BAM file (download from s3 + indexing + upload to S3) and set the timeout accordingly (perhaps with a 30% margin?).

Reply:
ok i'll do that now